docker/moonbase-parachain.Dockerfile

```diff
@@ -4,24 +4,24 @@
 
 FROM phusion/baseimage:0.11
 LABEL maintainer "alan@purestake.com"
-LABEL description="this is the parachain node running Moonbase Alphanet"
+LABEL description="this is the parachain node running Moonbase"
 ARG PROFILE=release
 
 RUN mv /usr/share/ca* /tmp && \
 	rm -rf /usr/share/*  && \
 	mv /tmp/ca-certificates /usr/share/ && \
 	rm -rf /usr/lib/python* && \
-	useradd -m -u 1000 -U -s /bin/sh -d /moonbase-alphanet moonbeam && \
-	mkdir -p /moonbase-alphanet/.local/share/moonbase-alphanet && \
-	chown -R moonbeam:moonbeam /moonbase-alphanet && \
-	ln -s /moonbase-alphanet/.local/share/moonbase-alphanet /data && \
+	useradd -m -u 1000 -U -s /bin/sh -d /moonbase-parachain moonbeam && \
+	mkdir -p /moonbase-parachain/.local/share/moonbase-parachain && \
+	chown -R moonbeam:moonbeam /moonbase-parachain && \
+	ln -s /moonbase-parachain/.local/share/moonbase-parachain /data && \
```

Comment:
Why is the directory still called `moonbase-parachain`? Why not just call it `moonbeam` or `data`.

Reply:
This docker image is for ops, I kept it as close as our current process